db/migrate/20190425000001_create_products.rb

```diff
@@ -5,8 +5,19 @@ def change
       t.text :description
       t.string :image_url
       t.decimal :price, precision: 8, scale: 2
+      t.boolean :enabled, :default => false
```

Comment:
@maskedXchaotic use the new syntax default: false

Reply:
done. used new syntax